src/main/kotlin/org/jetbrains/research/ml/codetracker/ui/panes/util/LanguagePane.kt

```diff
@@ -23,9 +23,10 @@ interface LanguageNotifier : Consumer<Int> {
 }
 
 open class LanguagePaneUiData : PaneUiData() {
-    val language = ListedUiField(
-        PluginServer.availableLanguages, 0,
-        LanguageNotifier.LANGUAGE_TOPIC, false)
+    companion object {
+        val language = ListedUiField(PluginServer.availableLanguages, 0,
+            LanguageNotifier.LANGUAGE_TOPIC, isRequired = false)
+    }
```

Comment:
After this change, I can not run the app :( It fails in **ReadOnlyDialogWrapper** class because the line `val currentLanguage = SurveyUiData.language.currentValue` has an error about getting **language** from **SurveyUiData**

Reply:
I guess my IDEA is broken since it doesn't tell me about such things :(